test/runner.sh

```diff
@@ -29,7 +29,7 @@ EXT_VERSION=${VERSION##version = }
 #docker doesn't set user
 USER=${USER:-`whoami`}
 
-TEST_SPINWAIT_ITERS=${TEST_SPINWAIT_ITERS:-10}
+TEST_SPINWAIT_ITERS=${TEST_SPINWAIT_ITERS:-100}
```

Comment:
Does it always wait for this amount of time, or does it complete sooner if the test succeeds?

Reply:
for successful tests this should not increase the test duration since it exits early if it succeeds this is just the maximum number of checks it does